experiments/compositions/composition/Makefile

```diff
@@ -152,6 +157,30 @@ docker-buildx: ## Build and push docker image for the manager for cross-platform
 	- $(CONTAINER_TOOL) buildx rm project-v3-builder
 	rm Dockerfile.cross
 
+##@ Testing
+
+.PHONY: release-kind-manifests
+release-kind-manifests: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
```

Comment:
Idea for the future: can we have one manifest?  Then when you test on kind, you build confidence on not-kind

Reply:
This is more of dev-testing since we auto fill the current project for the gcr images. 
Need a proper release target that generates manifests for images from release gcr + version

Misnamed. Release-devtest-manifests /?